scripts/xblock/list-installed.py

```diff
@@ -0,0 +1,31 @@
+"""
+Lookup list of installed XBlocks, to aid XBlock developers
+"""
+import pkg_resources
+
+
+def get_without_xmodules():
+    """
+    Get all installed XBlocks
+
+    but try to omit XModules, else the output is less helpful
```

Comment:
Nit: Could you please rephrase this from "XModules" to something like "built-in XBlocks"? I know the package still says XModule, but we've done the conversions to get rid of the XModule class usage, and I don't want to cause further confusion.

Reply:
@ormsbee Done; now referred to as built-in XBlocks.